CorsixTH/Src/th_gfx.h

```diff
@@ -484,129 +489,129 @@ class THAnimationManager
         @param iFirst First frame of the animation, or 0xFFFFFFFFu.
         @param iLength Number of frames in the animation.
      */
-    void fixNextFrame(uint32_t iFirst, size_t iLength);
+    void fix_next_frame(uint32_t iFirst, size_t iLength);
 };
 
 struct THMapNode;
-class THAnimationBase : public THDrawable
+class animation_base : public drawable
 {
 public:
-    THAnimationBase();
+    animation_base();
 
-    void removeFromTile();
-    void attachToTile(THMapNode *pMapNode, int layer);
+    void remove_from_tile();
+    void attach_to_tile(THMapNode *pMapNode, int layer);
 
-    uint32_t getFlags() const {return m_iFlags;}
-    int getX() const {return m_iX;}
-    int getY() const {return m_iY;}
+    uint32_t get_flags() const {return flags;}
+    int get_x() const {return x;}
+    int get_y() const {return y;}
 
-    void setFlags(uint32_t iFlags) {m_iFlags = iFlags;}
-    void setPosition(int iX, int iY) {m_iX = iX, m_iY = iY;}
-    void setLayer(int iLayer, int iId);
-    void setLayersFrom(const THAnimationBase *pSrc) {m_oLayers = pSrc->m_oLayers;}
+    void set_flags(uint32_t iFlags) {flags = iFlags;}
+    void set_position(int iX, int iY) {x = iX, y = iY;}
+    void set_layer(int iLayer, int iId);
+    void set_layers_from(const animation_base *pSrc) {layers = pSrc->layers;}
 
    // bool isMultipleFrameAnimation() { return false;}
 protected:
     //! X position on tile (not tile x-index)
-    int m_iX;
+    int x;
     //! Y position on tile (not tile y-index)
-    int m_iY;
+    int y;
```

Comment:
Perhaps a bit more expressing what the comment says, `xpos_on_tile` or `xpos_within_tile` or so?
Apparently this is currently confusing.

Reply:
In freerct, I used 256 sub-tile positions, multiplied the tile index by 256, and merged both numbers so I had a single position number that expressed the precise position in the game.